posts/2022-10-25-22.0.0.11.adoc

```diff
@@ -0,0 +1,440 @@
+* <<security_caches, Distributed security caches>>
+* <<bell, Expose SPI interfaces as BELL services and inject properties into BELL services>>
+* <<CVEs, Security Vulnerability (CVE) Fixes>>
+* <<bugs, Notable bug fixes>>
+
+
+View the list of fixed bugs in link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A220011+label%3A%22release+bug%22[22.0.0.11].
+
+Check out link:{url-prefix}/blog/?search=release&search!=beta[previous Open Liberty GA release blog posts].
+
+
+[#run]
+
+== Run your apps using 22.0.0.11
+
+If you're using link:{url-prefix}/guides/maven-intro.html[Maven], here are the coordinates:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.openliberty</groupId>
+    <artifactId>openliberty-runtime</artifactId>
+    <version>22.0.0.11</version>
+    <type>zip</type>
+</dependency>
+----
+
+Or for link:{url-prefix}/guides/gradle-intro.html[Gradle]:
+
+[source,gradle]
+----
+dependencies {
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[22.0.0.11,)'
+}
+----
+
+Or if you're using Docker:
+
+[source]
+----
+FROM open-liberty
+----
+
+Or take a look at our link:{url-prefix}/downloads/[Downloads page].
+
+[link=https://stackoverflow.com/tags/open-liberty]
+image::img/blog/blog_btn_stack.svg[Ask a question on Stack Overflow, align="center"]
+
+
+[#java19]
+== Java SE 19 Support
+
+Java SE 19 has been released and with it the following features:
+
+* link:https://openjdk.org/jeps/405[Record Patterns] (Preview)
+* link:https://openjdk.org/jeps/422[Linux/RISC-V Port]
+* link:https://openjdk.org/jeps/424[Foreign Functions & Memory API] (Preview)
+* link:https://openjdk.org/jeps/425[Virtual Threads] (Preview)
+* link:https://openjdk.org/jeps/426[Vector API] (Fourth Incubator)
+* link:https://openjdk.org/jeps/427[Pattern Matching for switch] (Third Preview)
+* link:https://openjdk.org/jeps/428[Structured Concurrency] (Incubator)
+
+Although Java SE 19 was not supported in 22.0.0.10 at the time of that version's release, it has been retroactively back-ported.  Therefore, Java SE 19 is officially supported with Open Liberty 22.0.0.10 and newer.  To give this a try today, link:https://adoptium.net/temurin/releases/?version=19[download Java SE 19], download and install at least the link:{url-prefix}/start/#runtime_releases[22.0.0.10] version of Open Liberty, edit your Liberty server's link:{url-prefix}/docs/latest/reference/config/server-configuration-overview.html#server-env[server.env file] to point JAVA_HOME to your Java SE 19 installation and start testing!
+
+For more information on Java SE 19, please visit the Java SE 19 link:https://jdk.java.net/19/release-notes[release notes page] or link:https://docs.oracle.com/en/java/javase/19/docs/api/index.html[API Javadoc page].
+For more information, refer to the link:{url-prefix}/docs/latest/java-se.html[Java SE support].
+
+NOTE: To try out preview features in Open Liberty, make sure to add `--enable-preview` to your `jvm.options` file.
```

Comment:
The way it is worded it makes it sound like you are trying Liberty preview features and not Java preview features.

```suggestion
NOTE: To try out Java EE 19 preview features in Open Liberty, make sure to add `--enable-preview` to your `jvm.options` file.
```

Reply:
I don't know why some of these suggestions are greyed out.... creating copies of them